dashboard/test/controllers/api/v1/users_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class Api::V1::UsersControllerTest < ActionController::TestCase
+  setup do
+    @user = create :user
+  end
```

Comment:
minor: you can get a small perf boost by using Will's recently-added `transactional_test_case` and `setup_all`. [Example](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb#L6-L7) to only create the user once.